src/components/RoomsList/RoomsListStyled.js

```diff
@@ -9,6 +29,29 @@ export const ListStyled = styled.div`
   width: 100%;
 `;
 
+export const Item = styled.li`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 72px;
+  padding: 0 5%;
+  background: ${({ $isActive }) =>
+    $isActive ? 'var(--color-blue-1)' : 'var(--white-color)'};
+  cursor: pointer;
+  transition: all 0.3s ease;
+
+  &:hover {
+    background: var(--color-blue-1);
+
+    & ${UnreadMessagesCount} {
+      background: var(--white-color);
+    }
+  }
+  @media ${device.tablet} {
+    padding: 0 16px;
+  }
+`;
+
 export const Text = styled.p`
   text-align: center;
   font-size: 14px;
```

Comment:
The same, we have it in the GlobalStyles.js.

Reply:
done